org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_detail.scss

```diff
@@ -65,11 +65,28 @@
 
   .crm-summary-block-wrap {
     width: 100%;
+    margin-top: 0;
   }
 
   #crm-contactname-content {
     border: none !important;
+    height: 60px;
     max-height: 60px;
+
+    .crm-summary-display_name {
+      padding-top: 22px;
+    }
+  }
+
+  .crm-summary-contactname-block {
+    display: flex;
+    flex-direction: row;
+    max-height: 120px;
```

Comment:
Magic number, please justify it. Also you are forcing a constraint on the height of the block, are we sure it's not gonna create a problem with the dynamic content?